ui/packages/consul-ui/mock-api/v1/acl/binding-rules

```diff
@@ -0,0 +1,30 @@
+[
+  ${
+    range(
+      env(
+        'CONSUL_BINDING_RULE_COUNT',
+        Math.floor(
+          (
+            Math.random() * env('CONSUL_BINDING_RULE_MAX', 10)
+          ) + parseInt(env('CONSUL_BINDING_RULE_MIN', 1))
+        )
+      )
+    ).map(
+      function(item, i) {
+        return `
+          {
+            "ID": "${fake.random.uuid()}",
+            "Description": "${fake.lorem.sentence()}",
+            "AuthMethod": "${fake.hacker.noun()}-${i}",
+            "Selector": "${fake.hacker.noun()}",
```

Comment:
Could we make the mocks respond with a realistic looking selector, you could probably just paste this one in and then use faker to fake the `default` and `vault` to give it a bit of randomness or something?

https://www.consul.io/api-docs/acl/binding-rules#selector

```
serviceaccount.namespace==default and serviceaccount.name!=vault
```